### PR TITLE
55034 lesson manager and api

### PR DESCRIPTION
**Pull request by jannatulabeda**
# Description
Lesson manager added and create lesson API implemented in Lesson view controller.

### CODE REVIEW

**Comment by hasanmainul** [comment]
LGTM

File: fels_111/LessonViewController.m [hunk 1]
```diff
@@ -60,14 +76,95 @@ -(void) setButtons {
...
+    Answer *answer = [word.answersArray objectAtIndex:0];
+    [self.answerButton1 setTitle:answer.answerContent forState:UIControlStateNormal];
+    
+    answer = [word.answersArray objectAtIndex:1];
+    [self.answerButton2 setTitle:answer.answerContent forState:UIControlStateNormal];
+    
+    answer = [word.answersArray objectAtIndex:2];
+    [self.answerButton3 setTitle:answer.answerContent forState:UIControlStateNormal];
+    
+    answer = [word.answersArray objectAtIndex:3];
```
Comment (TequilarTomsk) [inline fels_111/LessonViewController.m]: are you sure that the index is not over than array length?

File: fels_111/LessonViewController.m [hunk 2]
```diff
@@ -60,14 +76,95 @@ -(void) setButtons {
...
+    [self.answerButton4 setTitle:answer.answerContent forState:UIControlStateNormal];
+}
+
+#pragma mark - Button action
+- (IBAction)answerButtonPressed:(id)sender {
+    
+    UIButton *button = (UIButton *)sender;
+    Word *word = [self.wordsArray objectAtIndex:countWord-1];
+    
+    switch (button.tag) {
```
Comment (TequilarTomsk) [inline fels_111/LessonViewController.m]: redundant switch function here

File: fels_111/LessonViewController.m (hunk 1 above)
Comment (jannatulabeda) [inline fels_111/LessonViewController.m]: yes because server always sending 4 answers for each word

**Pull request merged**